docs/language-specification/structure/identifiers.md

```diff
@@ -0,0 +1,27 @@
+The first character of an identifier may either be an upper or lowercase letter from the ISO basic Latin alphabet or an underscore, _i.e._, it cannot be a digit.
+Any following character(s) follow the same rule as the first character, but now may also be a digit.
```

Comment:
```suggestion
```

Reply:
Why remove this?

Reply:
What about rephrasing the two sentences?

`An identifier is a sequence of any of the following characters: upper or lowercase letters from the ISO basic Latin alphabet, underscore, and digits. With the constraint that the first character cannot be a digit.`

Reply:
Can also be a single character though :P

Reply:
Then `An identifier is a sequence of one or more of the following characters: upper or lowercase letters from the ISO basic Latin alphabet, underscore, and digits. With the constraint that the first character cannot be a digit.`?